packages/react-core/src/hooks/__tests__/useGetUrl.test.ts

```diff
@@ -0,0 +1,206 @@
+import { renderHook } from '@testing-library/react-hooks';
+import * as Storage from 'aws-amplify/storage';
+
+import useGetUrl, { UseGetUrlInput } from '../useGetUrl';
+
+const getUrlSpy = jest.spyOn(Storage, 'getUrl');
+
+const url = new URL('https://amplify.s3.amazonaws.com/path/to/the/file.jpg');
+
+const onError = jest.fn();
+
+const KEY_INPUT: UseGetUrlInput = {
+  key: 'file.jpg',
+  options: { accessLevel: 'guest' },
+  onError: onError,
+};
+
+const PATH_INPUT: UseGetUrlInput = {
+  path: 'guest/file.jpg',
+  onError: onError,
```

Comment:
Prefer object short hand:
```suggestion
  onError,
```